src/pytest_servers/fixtures.py

```diff
@@ -14,16 +18,16 @@ def aws_region_name():
 
 @pytest.fixture
 def s3_server(monkeypatch):
-    """Spins up a moto s3 server on 127.0.0.1:5555."""
+    """Spins up a moto s3 server on a random port."""
     monkeypatch.setenv("AWS_SECRET_ACCESS_KEY", "foo")
     monkeypatch.setenv("AWS_ACCESS_KEY_ID", "foo")
-    with MockedS3Server() as server:
+    with MockedS3Server(random.randint(60000, 65535)) as server:
         yield server
```

Comment:
In dvc we use one single moto server (to reflect single real s3 server) and use tmp paths within it. Spamming ports with a fixture of such a low scope might become problematic with a growing number of tests.